agate-webapp/src/main/resources/i18n/en.json

```diff
@@ -535,6 +535,8 @@
     "new-realm": "New Realm",
     "users-help": "List of users authenticated by this realm.",
     "users-none": "No users are authenticated by this realm.",
+    "username-claim": "Username Claim",
+    "username-claim-help": "Optional key used to get the username from the JWT custom claims. If not found, 'preferred_username' will be used as username claim.",
```

Comment:
Not quite... See Opal equivalent:

https://github.com/obiba/opal/blob/master/opal-gwt-client/src/main/java/org/obiba/opal/web/gwt/app/client/administration/idproviders/edit/IDProviderView.ui.xml#L144-L147

Reply:
I am not sure how good is the FR translation, maybe you could suggest a better one?

Reply:
Effectivement, la traduction de claim n'est pas très heureuse... Peut-être utiliser "paramètre" pour faire simple?